src/main.py

```diff
@@ -0,0 +1,108 @@
+from src import data_handler as dh
+from src import activity_tracker_handler as ath
+from src import consts
+import pandas as pd
+import logging
+import csv
+import sys
+import os
+
+pd.set_option('display.max_rows', 250)
+pd.set_option('display.max_columns', 100)
+
+
+def __get_data_path():
+    args = sys.argv
+    if len(args) < 3 or consts.PATH_CMD_ARG not in args:
+        raise NameError('List of arguments must have required 1 element: path!')
+    try:
+        path = args[args.index(consts.PATH_CMD_ARG) + 1]
+    except ValueError:
+        raise NameError('List of arguments does not have the argument -path')
+    except IndexError:
+        raise NameError('List of arguments does not have a value after the argument -path')
+    return path
+
+
+def __count_containing_substring(list_of_string: list, substring: str):
+    count = 0
+    for i, s in enumerate(list_of_string):
+        if substring in s:
+            count += 1
+    return count
+
+
+def __index_containing_substring(list_of_string: list, substring: str):
+    for i, s in enumerate(list_of_string):
+        if substring in s:
+            return i
+    return -1
+
+
+def __get_real_ati_file_index(path: str, files: list, ati_key: str):
+    sniffer = csv.Sniffer()
+    sample_bytes = 1024
+    for i, f in enumerate(files):
+        if ati_key in f:
+            if not sniffer.has_header(open(path + '/' + f, encoding=consts.ENCODING).read(sample_bytes)):
+                return i
+    return -1
+
+
+def __separate_ati_and_other_files(files: list, folder: str, full_path: str):
+    count_ati = __count_containing_substring(files, consts.ACTIVITY_TRACKER_FILE_NAME)
+    ati_file = None
+    ati_id = None
+    if count_ati > 1:
+        ati_file_index = __get_real_ati_file_index(full_path, files, consts.ACTIVITY_TRACKER_FILE_NAME)
+    else:
+        ati_file_index = __index_containing_substring(files, consts.ACTIVITY_TRACKER_FILE_NAME)
```

Comment:
There are other arguments of changing the __get_real_ati_file_index function and removing __count_containing_substring and __index_containing_substring:
1. in this implementation in case of only one file with AT name, this file isn't checked for having headers. For example,
there may be a CT file with events of editing an AT file, so the name of the CT file may contain AT name. 
2.  in this implementation you are iterating twice over all files (first by counting files containing AT name, second by calling  __get_real_ati_file_index or __index_containing_substring). You can avoid it by calling the changed __get_real_ati_file_index function once.